WooCommerce/Classes/ViewRelated/Dashboard/Settings/Plugins/PluginListViewController.swift

```diff
@@ -17,6 +19,59 @@ final class PluginListViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        configureNavigation()
+        configureTableView()
+        configureViewModel()
+    }
+}
+
+// MARK: - UI Configurations
+private extension PluginListViewController {
+    func configureNavigation() {
+        title = NSLocalizedString("Plugins", comment: "Title of the Plugin List screen")
+    }
+
+    func configureTableView() {
+        tableView.registerNib(for: HeadlineLabelTableViewCell.self)
+        tableView.estimatedRowHeight = CGFloat(44)
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.backgroundColor = .listBackground
+        tableView.allowsSelection = false
+        tableView.dataSource = self
+    }
+
+    func configureViewModel() {
+        viewModel.observePlugins { [weak self] in
+            self?.tableView.reloadData()
+        }
+    }
+}
+
+// MARK: - UITableViewDataSource conformance
+//
+extension PluginListViewController: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        viewModel.numberOfSections
     }
 
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        viewModel.numberOfRows(inSection: section)
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(HeadlineLabelTableViewCell.self, for: indexPath)
+        let cellModel = viewModel.cellModelForRow(at: indexPath)
+        cell.update(style: .bodyWithLineLimit(count: 2),
+                    headline: cellModel.name,
+                    body: cellModel.description)
```

Comment:
nit: I think this can go in one line. We normally spawn over multiple lines when going beyond the `swiftlint` limit.